themes/bootstrap/static/js/color-console-output.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+let html = document.querySelector('section').innerHTML;
+
+// Find the ANSI escape sequences for coloring the text
+// const ansiPatterns = [/\^\[\[\d*m/g, /\^\[\[\d*;\d*m/g, /\^\[\[\d*;\d*;\d*m/g];
+// for (let pattern of ansiPatterns) {
+//   console.log(new Set(html.match(pattern)));  // Used in ansiSequencesToSpans
+// }
+
+// Manually format the relevant patterns
+const ansiSequencesToSpans = {
+  // '\\^\\[\\[0m': '<span>',
+  '\\^\\[\\[4m': '<span style="text-decoration:underline;">',
+  '\\^\\[\\[37m': '<span>',
+  '\\^\\[\\[37;44m': '<span style="background-color:tomato">',
```

Comment:
Just food for thought: Do you think it would be possible rather than match predefined patterns to parse the escape sequence and to translate this into a CSS color? 🤔 
E.g. One could restrict the supported color sequences for background and foreground to [24-bit](https://en.wikipedia.org/wiki/ANSI_escape_code#24-bit):
```
ESC[38;2;⟨r⟩;⟨g⟩;⟨b⟩ m Select RGB foreground color
ESC[48;2;⟨r⟩;⟨g⟩;⟨b⟩ m Select RGB background color
```
which then could be translated to CSS using `rgb(<r>, <g>, <b>)`... 🤔 

Somehow I feel I am starting to design a highlight.js plugin now 😅

Reply:
Will do. Agreed that my "provisional" solution requires too much manual intervention. By parsing the ANSI codes like this, the remaining docs pages will be easy to transcribe too. Thank you for the feedback.